<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PROYECTO1</class>
 <widget class="QMainWindow" name="PROYECTO1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PROYECTO1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>571</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="BtnLogin">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnCrearProyecto">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Proyecto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnPublicar">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Publicar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnImportar">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Importar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnNuevo">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Nuevo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnAbrir">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSave">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSaveAs">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar Como</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnCerrar">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnTraducir">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>280</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Traducir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnReportes">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reportes</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Consola">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>711</width>
      <height>131</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineaComandos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>711</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
